procurement/tests/test_email_alerts.py

```diff
@@ -45,7 +45,7 @@ def setUp(self):
             slug="shropshire",
             authority_code="SHR",
             gss_code="E06000051",
-            nation="Shropshire",
+            nation="England",
```

Comment:
Rather than leave this as single commit it's slightly nicer from a history perspective to rebase the branch and include it in the original commit.

Reply:
Makes sense!